controllers/operands/monitoring.go

```diff
@@ -241,6 +243,20 @@ func NewPrometheusRuleSpec() *monitoringv1.PrometheusRuleSpec {
 						componentAlertLabelKey: componentAlertLabelValue,
 					},
 				},
+				{
+					Alert: installationNotCompletedAlert,
+					Expr:  intstr.FromString("(1 - kubevirt_hco_hyperconverged_cr_exists) > 0"),
+					Annotations: map[string]string{
+						"description": "the installation was not completed; the HyperConverged custom resource is missing.",
```

Comment:
@nunnatsa perhaps, 
In order to complete the installation of the Hyperconverged Cluster Operator you should create the HyperConverged custom resource.

On d/s runbooks we can rename to OpenShift Virtualization.